src/js/components/AppListComponent.jsx

```diff
@@ -222,17 +223,17 @@ var AppListComponent = React.createClass({
           </tr>
           <tr className={errorClassSet}>
             <td className="text-center text-danger" colSpan="6">
-              Error fetching apps. Refresh to try again.
+              Error fetching apps. {Messages.RETRY_REFRESH}
```

Comment:
The problem with mixing text and expressions inside JSX is that React will need to wrap each part in a `<span>` element. It's definitely not a tragedy, but it might potentially apply unwanted CSS to our markup.



<img width="816" alt="screen shot 2015-10-06 at 10 58 05" src="https://cloud.githubusercontent.com/assets/1078545/10304228/25c24ec6-6c19-11e5-8d4d-eb5ef1025d96.png">

The cleanest solution to this is by doing:

```
<tr>{`Error fetching apps. ${Messages.RETRY_REFRESH}`}</tr>
```